main.js

```diff
@@ -0,0 +1,106 @@
+/* top menu */
+const ham = document.querySelector('.hamburger');
+const menu = document.querySelector('.menu');
+const menuItem = document.querySelectorAll('.menu-items a');
+
+ham.addEventListener('click', () => {
+  if (ham.classList.contains('active')) {
+    ham.classList.remove('active');
+    menu.style.transition = 'all 0.1s ease-out';
+    menu.style.left = '-500%';
+  } else {
+    ham.classList.add('active');
+    menu.style.transition = 'all 0.3s ease-in';
+    menu.style.left = '0';
+  }
+});
+
+menuItem.forEach((link) => {
+  if (ham.classList.contains('active')) {
+    link.addEventListener('click', () => {
+      ham.classList.remove('active');
+      menu.style.left = '-500%';
+    });
+  } else {
+    link.addEventListener('click', () => {
+      ham.classList.remove('active');
+      menu.style.left = '-500%';
+    });
+  }
+});
+
+/* speakers section */
+
+const speakers = [
+  {
+    name: 'Bert Blevins',
+    picture: './icons/a.jpg',
+    occupation: 'Successful student',
+    description: 'Greetings to everyone at Blockchain Council! I recently completed my course on Blockchain from the council. I’m especially impressed with the quality of your content...',
+  },
+  {
+    name: 'Ahmed Yousry AbouElDahab',
```

Comment:
- [ ] Please add the dotted background image it is shown in the original design on each card of the speakers' section.
![Screenshot (3)](https://user-images.githubusercontent.com/47625407/182980295-e23b0e0e-deb2-44c8-a880-9592c8b28d8a.png)